### PR TITLE
Feature/error catching

### PR DESCRIPTION
**Pull request by souyahia-monk**

### CODE REVIEW

File: packages/toolkit/src/hooks/useError/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { useCallback } from 'react';
+
+export default function useError() {
+  return useCallback((error) => {
+    console.error('Error detected !!!!!!');
```
Comment (alexandre-em) [inline packages/toolkit/src/hooks/useError/index.js:5]: I don't think we will put the sentry on the sdk, so perhaps we can remove this hook.. what do you think?

Comment (natsukiai) [reply packages/toolkit/src/hooks/useError/index.js:5]: We could put Sentry on the SDK with an Opt-in param and a dedicated dsn for customers.

**Review by natsukiai — COMMENTED** [review]
Just need a console.log() clean up

**Review by natsukiai — CHANGES_REQUESTED** [review]
Fix eslint errors

**Comment by natsukiai** [comment]
@souyahia-monk can we remove or add eslint ignores for the `console.error` making _github_ code scan throw a warning in the file changes ?

**Pull request merged**